File: IN1010/trix/go/butikk/person.go
package main

type Person struct {
	navn string
	gjenstand string
	neste *Person
}

func (p *Person) HentNavn() string {
	return p.navn
}

func (p *Person) HentGjenstand() string {
	return p.gjenstand
}

func (p *Person) EndreNeste(ny *Person) {
	p.neste = ny
}

func (p *Person) HentNeste() *Person {
	return p.neste
}

File: IN1010/trix/06/butikk/Person.java
class Person {
    String navn;
    String gjenstand;
    Person neste;

    public Person(String navn, String gjenstand, Person neste) {
        this.navn = navn;
        this.gjenstand = gjenstand;
        this.neste = neste;
    }

    public void SettNeste(Person neste) {
        this.neste = neste;
    }

    public String HentNavn() {
        return this.navn;
    }

    public String HentGjenstand() {
        return this.gjenstand;
    }

    public Person HentNeste() {
        return this.neste;
    }
}
